mayastor/src/core/env.rs

```diff
@@ -199,6 +213,9 @@ impl Default for MayastorEnvironment {
     fn default() -> Self {
         Self {
             config: None,
+            enable_grpc: false,
```

Comment:
This cannot be optional. grpc server must run as control plane depends on it. Or is it something you plan to change once mayastor grpc code is removed from ms agent?

Reply:
@jkryl this actually more complicated than just that. For example, the nvmf initiator should not start the gRPC server when we are running this during the tests. Does that make sense to you?

Reply:
AFAIK our initiator utility for testing iscsi and nvmf targets does not use core::env. mayastor seems to be the only one using this code. Though if it makes more sense to leave this optional and hardcode enable_grpc parameter in mayastor code that creates the env, then that is also possible.

Reply:
The initiator utility *does* use core::env

Reply:
Indeed, initially, it did not, but it was changed 2 months ago. So I agree that the current approach makes sense then 👍